locales/en.json

```diff
@@ -237,5 +236,20 @@
     "yunohost_ca_creation_failed": "Unable to create certificate authority",
     "yunohost_configured": "YunoHost has been configured",
     "yunohost_installing": "Installing YunoHost...",
-    "yunohost_not_installed": "YunoHost is not or not correctly installed. Please execute 'yunohost tools postinstall'."
+    "yunohost_not_installed": "YunoHost is not or not correctly installed. Please execute 'yunohost tools postinstall'.",
```

Comment:
oh ok I now understand the sentence... bad wording imho, but I understand this was present before your PR